.github/workflows/release-publish-package.yml

```
@@ -0,0 +1,27 @@
+name: "Publish Immutable Action Version"
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+  
+    permissions:
+      contents: read
+      id-token: write
+      packages: write
+
+    steps:
+      - uses: actions/checkout@v4
+      
+      - name: Publish
+        id: publish
+        uses: actions/publish-immutable-action@0.0.1
```

```suggestion
        uses: actions/publish-immutable-action@0.0.1
```

The readme suggests using @0.0.3

https://github.com/actions/publish-immutable-action?tab=readme-ov-file#usage